DiamondLightSource/hyperion
# Add managing the xbpm feedback in xray centring

Fixes #796

Link to dodal PR (if required): https://github.com/DiamondLightSource/dodal/pull/160

### To test:
1. Confirm tests still pass
2. Confirm documentation on why we need to change xbpm makes sense
